Tribler/Test/test_as_server.py

```diff
@@ -91,6 +92,36 @@ def temporary_directory(self, suffix='', exist_ok=False):
                 raise e
         return temp
 
+    def get_bucket_range_port(self):
+        """
+        Return the port range of the test bucket assigned.
+        """
+        test_bucket = os.environ.get("TEST_BUCKET", None)
+        if test_bucket is not None:
+            test_bucket = int(test_bucket) + int(os.environ.get('TEST_BUCKET_OFFSET', 0))
+
+        min_base_port = 1000 if test_bucket is None else test_bucket * 2000 + 2000
```

Comment:
Ports 0-1023 are system ports and are unavailable to users. So, `min_base_port` should be at least 1024